### PR TITLE
Frontend role

### PR DESCRIPTION
**Pull request by longhotsummer**
@nefiger

### CODE REVIEW

File: careers.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h1 class="text-center header">Careers</h1>
 	<section class="current-careers">
 		<div class="container">
 
-		<h2>Interested in working for OpenUp?</h2>
+		<h2>Interested in working with OpenUp?</h2>
```
Comment (adieyal) [inline careers.html]: At open up?

Comment (longhotsummer) [inline careers.html]: Under? Above? INSIDE?

Comment (longhotsummer) [inline careers.html]: I can work with "at", feels better than "for". :)

File: careers.html [hunk 2]
```diff
@@ -48,71 +43,36 @@ <h1 class="text-center header">Careers</h1>
...
-							<li>Experience writing data scrapers</li>
-							<li>A knowledge of the Unix command line, including various utilities such as grep, awk and sed</li>
-							<li>A knowledge of Javascript d3js for visualisation</li>
-							<li>Experience working with online mapping tools such as CartoDB as well as QGIS</li>
-							<li>A basic understanding of SVG</li>
-							<li>A solid working knowledge of Excel or equivalent software</li>
+							<li>At least 3 years' experience building web applications and frontends</li>
+							<li>Solid knowledge of Javascript, jQuery, CSS3 and HTML5</li>
+							<li>Solid knowledge of at least some of the modern frontend technologies, such as Angular, Backbone, React, Handlebars, Underscore, D3, Vue, etc.</li>
+							<li>Experince implementing frontends using a REST API</li>
```
Comment (adieyal) [inline careers.html]: Experience

File: careers.html [hunk 3]
```diff
@@ -48,71 +43,36 @@ <h1 class="text-center header">Careers</h1>
...
+							<li>Experince using Agile processes in a small team</li>
+							<li>Familiarity with GitHub and GitHub Pages</li>
 							<li>Has their own laptop</li>
-							<li>Has a curious mind and loves playing with datasets</li>
+							<li>Permission to work full-time in South Africa</li>
```
Comment (adieyal) [inline careers.html]: Non-South Africans must have a valid work permit

File: careers.html (hunk 2 above)
Comment (adieyal) [inline careers.html]: Experience

File: careers.html [hunk 4]
```diff
@@ -48,71 +43,36 @@ <h1 class="text-center header">Careers</h1>
...
-						<p class="career-desc">The position is based in Cape Town but some assignments will be based elsewhere for short periods of time.</p>
+						<p class="career-desc">We’re looking for an intermediate, experience frontend developer to help us build useful, intuitive, open-source websites and tools.</p>
+						<p class="career-desc"><a href="/tools/">Our projects</a> range from small, single-page embeddable tools to large websites with rich content. The succesful applicant will help design and implement these projects and ensure they're helpful and user-friendly.</p>
+
+						<p class="career-desc">Candidates should be well-versed in modern frontend development tools and best-practices, be strong advocates for a good user experience, and able to work well independently and collaboratively.</p>
+						<p class="career-desc">We're particularly interested in candidates that share our passion for working to help South Africans make more informed decisions.</p>
+						<p class="career-desc">The position is based in Cape Town.</p>
```
Comment (adieyal) [inline careers.html]: full-time and based in Cape Town

File: careers.html [hunk 5]
```diff
@@ -48,71 +43,36 @@ <h1 class="text-center header">Careers</h1>
...
-						<p class="career-desc">The position is based in Cape Town but some assignments will be based elsewhere for short periods of time.</p>
+						<p class="career-desc">We’re looking for an intermediate, experience frontend developer to help us build useful, intuitive, open-source websites and tools.</p>
+						<p class="career-desc"><a href="/tools/">Our projects</a> range from small, single-page embeddable tools to large websites with rich content. The succesful applicant will help design and implement these projects and ensure they're helpful and user-friendly.</p>
+
+						<p class="career-desc">Candidates should be well-versed in modern frontend development tools and best-practices, be strong advocates for a good user experience, and able to work well independently and collaboratively.</p>
+						<p class="career-desc">We're particularly interested in candidates that share our passion for working to help South Africans make more informed decisions.</p>
```
Comment (adieyal) [inline careers.html]: I realise more and more that I am completely out of touch with what drives developers. One technical challenge that may appeal to a more senior intermediate is the ability to conceptualise how to maximise the value that users get from a site.

File: careers.html [hunk 6]
```diff
@@ -48,71 +43,36 @@ <h1 class="text-center header">Careers</h1>
...
-						<p class="career-desc">The candidate requires general scripting skills in python although extensive experience developing enterprise systems is not expected. The position will require some travel and the ability to work on intensive projects on tight deadlines. Projects will be relatively short and the candidate will be need to be creative about solving problems. He or she should also be confident in presenting their work to audiences.</p>
-						<p class="career-desc">The position is based in Cape Town but some assignments will be based elsewhere for short periods of time.</p>
+						<p class="career-desc">We’re looking for an intermediate, experience frontend developer to help us build useful, intuitive, open-source websites and tools.</p>
+						<p class="career-desc"><a href="/tools/">Our projects</a> range from small, single-page embeddable tools to large websites with rich content. The succesful applicant will help design and implement these projects and ensure they're helpful and user-friendly.</p>
```
Comment (adieyal) [inline careers.html]: helpful and user-friend is a little bland - how about something around engaging users, defining and achieving goals, maybe taking a more analytical approach to understanding what users are doing on the site - just some thoughts

File: careers.html [hunk 7]
```diff
@@ -48,71 +43,36 @@ <h1 class="text-center header">Careers</h1>
...
-						<p class="career-desc">The candidate requires general scripting skills in python although extensive experience developing enterprise systems is not expected. The position will require some travel and the ability to work on intensive projects on tight deadlines. Projects will be relatively short and the candidate will be need to be creative about solving problems. He or she should also be confident in presenting their work to audiences.</p>
-						<p class="career-desc">The position is based in Cape Town but some assignments will be based elsewhere for short periods of time.</p>
+						<p class="career-desc">We’re looking for an intermediate, experience frontend developer to help us build useful, intuitive, open-source websites and tools.</p>
```
Comment (adieyal) [inline careers.html]: is the word experience necessary?

File: careers.html (hunk 1 above)
Comment (adieyal) [inline careers.html]: totes

**Pull request merged**